pkg/ocrtypecodec/v1/exec.go

```diff
@@ -79,14 +79,20 @@ func (e *ExecCodecProto) DecodeObservation(data []byte) (exectypes.Observation,
 }
 
 func (e *ExecCodecProto) EncodeOutcome(outcome exectypes.Outcome) ([]byte, error) {
-	outcome = exectypes.NewSortedOutcome(outcome.State, outcome.CommitReports, outcome.Report)
+	outcome = exectypes.NewOutcome(outcome.State, outcome.CommitReports, outcome.Reports)
 
 	pbObs := &ocrtypecodecpb.ExecOutcome{
-		PluginState:   string(outcome.State),
-		CommitReports: e.tr.commitDataSliceToProto(outcome.CommitReports),
-		ExecutePluginReport: &ocrtypecodecpb.ExecutePluginReport{
-			ChainReports: e.tr.chainReportsToProto(outcome.Report.ChainReports),
-		},
+		PluginState:          string(outcome.State),
+		CommitReports:        e.tr.commitDataSliceToProto(outcome.CommitReports),
+		ExecutePluginReports: e.tr.execPluginReportsToProto(outcome.Reports),
```

Comment:
Hmm actually not sure if you can do this, it'll break oracles on the previous version since they won't know how to decode this field, and they'll just get an empty `ExecutePluginReport`